tests/test_magnification_dialog.py

```diff
@@ -48,6 +52,8 @@ def test_magnification_ui(
     - Magnification is propagated to the table when "ok" is clicked.
     """
     dlg = cpt_widget._on_click_magnification()
+    if double_click:
+        dlg = cpt_widget._on_click_magnification()
```

Comment:
```suggestion
    # The user opens the child dialog (also test fringe case that they click the button again).
    if click_twice:
        cpt_widget._on_click_magnification()
    dlg = cpt_widget._on_click_magnification()
```
Also.. can we use the full word `dialog` 🥺. It's only three extra characters.